### DISCUSSION TITLE
git tag "2.0.0"

### DISCUSSION THREAD

**Discussion by itkfm** [kimai/kimai#3840] [General]
Has the current [`2.0.0` tag](https://github.com/kimai/kimai/releases/tag/2.0.0) been pushed on purpose?
Or will it get removed later once again?

What’s the current state of v2.0.0?

<details>

<summary>I’m asking because said tag has been re-assigned apparently…</summary>

```sh
git fetch --tags
remote: Enumerating objects: 747, done.
remote: Counting objects: 100% (410/410), done.
remote: Compressing objects: 100% (90/90), done.
remote: Total 151 (delta 129), reused 76 (delta 61), pack-reused 0
Receiving objects: 100% (151/151), 76.52 KiB | 932.00 KiB/s, done.
Resolving deltas: 100% (129/129), completed with 72 local objects.
From https://github.com/kimai/kimai
 * [new tag]           1.30.6       -> 1.30.6
 * [new tag]           1.30.7       -> 1.30.7
 ! [rejected]          2.0.0        -> 2.0.0  (would clobber existing tag)
 * [new tag]           2.0.0-beta-3 -> 2.0.0-beta-3
 * [new tag]           2.0.0-rc-1   -> 2.0.0-rc-1
 ```

</details>

**Comment by kevinpapst** [comment]
It was accidentally pushed a while ago and was available for probably 1 hour.
So nothing to worry about, you might be the only one who fetched it previously.

**Reply by itkfm** [reply]
Thank you for the explanation :)
